### PR TITLE
feat(wmg): Users need a way to assess the strength of marker genes

### PR DESCRIPTION
**Pull request by ashin-czi**
## Reason for Change

- #4087
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- add
    - Update tool tip to show the range of scores 
- remove
- modify

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

### CODE REVIEW

**Comment by ashin-czi** [comment]
image

The initial layout, still waiting for figma

**Comment by ashin-czi** [comment]
How does this look @karenliang0 @pablo-gar 
https://user-images.githubusercontent.com/109984998/220420426-e38fe2c6-2086-4b66-8e17-324a1a661d8a.mov

Still wondering if we were going to still remove the bad marker genes and limit it to top 25 per the Slack converation?

**Comment by ashin-czi** [comment]
removal of bad genes will be another PR

File: frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx [hunk 1]
```diff
@@ -206,6 +197,26 @@ function CellInfoSideBar({
...
+                onClick={handleCopyGenes}
+                sdsType="primary"
+                sdsStyle="minimal"
+                isAllCaps={false}
+                startIcon={<Icon sdsIcon="copy" sdsSize="s" sdsType="button" />}
+              >
+                Copy
+              </CopyGenesButton>
+            </td>
+            <td style={{ verticalAlign: "middle" }}>
```
Comment (seve) [inline frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx:212]: Can you switch this to `emotion.styled`

**Review by seve — APPROVED** [review]

**Pull request merged**